docs/phdata.rst

```diff
@@ -232,16 +261,33 @@ beam-splitter the fields:
 
 - **split_ch1**
 - **split_ch2**
+- etc..
 
 specify which detector pixel is used in each of the "beam-split" channels.
 
-All previous fields are arrays containing one or more :ref:`detector_ids`.
-For example, a 2-color smFRET measurement will have only one value in
-``spectral_ch1`` (donor) and one value in ``spectral_ch2``
-(acceptor). A 2-color smFRET measurement with polarization
+If :ref:`marker_ids` are also recorded, then the fields (*new in v0.5*):
+
+- **markers1**
+- **markers2**
```

Comment:
Bikeshedding the names.

- Might it be more uniform to use the `_ch1`, `_ch2` suffixes to indicate that that these contain the same types of ids as `spectral_ch1`, `split_ch1`, etc? (Counterargument would be that "channel" is to be used only for detectors that observe something split from a common photon source. If so, perhaps this should be defined explicitly.)

- Having thought about this a bit more, if the intent is to use this as a mechanism to segregate out all timestamps that are not photons (including things like a laser SYNC or raw (pre-CFD) photon falling/rising edges recorded by Swabian Time Tagger), it might be better to use a more direct name such as `non_photon_ch1`, `non_photon_ch2`, etc., because "marker" has a fairly specific connotation (at least to those familiar with PQ and BH data). We probably don't want to keep adding categories that a photon-only reader needs to skip.

Reply:
- ``_ch1`` ``_ch1`` I have no strong thoughts one way or another, I've though the same as you that having ``_ch1`` ``_ch2`` serves as a way to set apart enumerations of IDs, but it also suggests they are photons
- I think I agree with your thoughts here, I would agree ``non_photon`` is a better name.

Reply:
I've been giving the naming some more thought, and the reality is there are at least two distinct categories of non-photon detectors, those that correspond to some form of detected event, ie a sync signal, a Swabian rising/falling edge tag etc. and those that are produced internally by the software, ie markers indicating the advance of a pixel or a line.

I'm looking to 0.6 when we introduce a FLIM standard, how will we define which ``non_photon/marker`` channels correspond to which marker (pixel/line advance). The ``/setup/scan/`` group could contain a mapping, but it would be less intuitive for a person reading the file. Another strategy might be to say that ```detectors_specs/non_photonX`` means a detector with a purpose defined in the ``/user/`` group, and in 0.6 we will introduce specific fields ``detectors_specs/scan_chX`` and maybe ``detectors_specs/sync_chX`` to more concretely define functions of specific categories of non-photon events.